python/mxnet/optimizer.py

```diff
@@ -541,25 +546,27 @@ class RMSProp(Optimizer):
     If ``centered=False``, we follow
     http://www.cs.toronto.edu/~tijmen/csc321/slides/lecture_slides_lec6.pdf by
     Tieleman & Hinton, 2012.
+    For details of the update algorithm see :class:`~mxnet.ndarray.rmsprop_update`
 
     If ``centered=True``, we follow http://arxiv.org/pdf/1308.0850v5.pdf (38)-(45)
     by Alex Graves, 2013.
+    For details of the update algorithm see :class:`~mxnet.ndarray.rmspropalex_update`
 
     This optimizer accepts the following parameters in addition to those accepted
     by :class:`.Optimizer`:
 
     Parameters
     ----------
     gamma1: float, optional
-        Decay factor of moving average for ``gradient^2``.
+        A decay factor of moving average over past squared gradient.
     gamma2: float, optional
-        A "momentum" factor. Only used if ``centered=True``.
+        A "momentum" factor. Only used if `centered`=``True``.
     epsilon : float, optional
-        Small value to avoid division by 0.
+        Small value to avoid divided by 0.
     centered : bool, optional
-        Use Graves' or Tieleman & Hinton's version of RMSProp.
+        Uses Graves's version of `RMSProp` or Tieleman & Hinton's version of `RMSProp`.
```

Comment:
you should reword this to say:
Flag to control which version of RMSProp to use. ``True`` will use X, ``False`` will use Y